File: 1448-maximum-69-number/maximum-69-number.java
class Solution {
    public int maximum69Number(int num) {
        

        for (int i = 3; i >= 0; i--) {
            int n = (int) (num / Math.pow(10, i));
            if (n % 10 == 6) {
                num += (int) 3 * Math.pow(10, i);
                break;
            }
        }

        return num;
    }
}

File: 1448-maximum-69-number/maximum-69-number.go
func maximum69Number (num int) int {
    numStr := []rune(strconv.Itoa(num))
	for i := 0; i < len(numStr); i++ {
		if numStr[i] == '6' {
			numStr[i] = '9'
			break
		}
	}
	result, _ := strconv.Atoi(string(numStr))
	return result
}